core/config/config_loader.py

```
@@ -0,0 +1,451 @@
+    def load_environment(self, prefix: str = "HADES_") -> Dict[str, Any]:
+        """
+        Load configuration from environment variables.
+
+        Args:
+            prefix: Environment variable prefix
+
+        Returns:
+            Configuration data from environment
+        """
+        config: Dict[str, Any] = {}
+        prefix_lower = prefix.lower()
```

Ruff (0.14.13)

Local variable `prefix_lower` is assigned to but never used

Remove assignment to unused variable `prefix_lower`

(F841)